afutests/memcpy/README.md

```diff
@@ -25,6 +25,7 @@ Usage
         -i            Send an interrupt after copy
         -l <loops>    Run this number of memcpy loops (default 1)
         -p <procs>    Fork this number of processes (default 1)
+                      Use -p0 to fork as many of processes as permitted by AFU
```

Comment:
I think we could improve the wording here, maybe "Use -p0 the maximum number of processes permitted by the AFU"

Reply:
I am not sure that we should make the maximum value visible to user, because this value is likely to change in future implementations of memcpy, or future revisions of OpenCAPI.

I have precisely chosen -p0 to select the maximum value regardless of the AFU implementation. Testing with 'ocxl_memcpy -p0' should alway stress the AFU to its current limit.

Reply:
Ok, we could drop the (512) from above then, but we should still rewrite it as it reads better.